less/admin/UsersListPage.less

```diff
@@ -0,0 +1,34 @@
+.UsersListPage {
+  &-grid {
+    width: 100%;
+
+    &--loaded {
+      display: grid;
+      overflow-x: auto;
+    }
+
+    &--header {
+      font-weight: bold;
+      border-bottom: 1px solid #999;
```

Comment:
These colors should use less variables for css / theming compatibility

Reply:
Done!